proto/usr/share/initramfs-tools/scripts/live-bottom/triton-agents

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2021 Joyent, Inc.
+#
+
+#
+# This script is responsible for setting up the Triton agents.
+#
+
+PREREQ="triton-general"
+
+# Pre-script requirements.
+if [ "$1" = "prereqs" ]; then
+    echo "${PREREQ}"
+	exit 0
+fi
+
+# shellcheck disable=SC1091
+. /scripts/functions
+# shellcheck source=proto/usr/share/initramfs-tools/scripts/triton
+. /scripts/triton
+
+#set -o xtrace
+
+log_begin_msg "triton-agents"
+
+# Simply calling this creates necessary directories
+# shellcheck disable=SC2034
+triton_config_dir=$(getTritonConfigDir)
+
+rootmnt_agent_node_dir="${rootmnt:-/}/opt/smartdc/agents/lib/node_modules"
+# usr_triton_agents_dir="/usr/triton/defaults/agents"
```

Comment:
Needed or can we delete?

Reply:
Not needed. That var wasn't used anywhere so shellcheck was complaining about it. I commented it out so I could put it back if something broke. It's been out a while and everything seems ok still.